backpack/extensions/firstorder/batch_l2_grad/convnd.py

```diff
@@ -1,23 +1,79 @@
+"""batch_l2 extension for Conv."""
 from torch import einsum
 
-from backpack.extensions.firstorder.base import FirstOrderModuleExtension
+from backpack.extensions.firstorder.batch_l2_grad.batch_l2_base import BatchL2Base
 from backpack.utils import conv as convUtils
 
 
-class BatchL2ConvND(FirstOrderModuleExtension):
+class BatchL2ConvND(BatchL2Base):
+    """batch_l2 extension for Conv."""
+
     def __init__(self, N, params=None):
+        """Initialization.
+
+        Args:
+            N: number of dimensions
+            params: list of parameter names. Defaults to None.
+        """
         super().__init__(params=params)
         self.N = N
 
-    # TODO Use bias Jacobian to compute `bias_gradient`
```

Comment:
Correct to delete TODO?

Reply:
Correct. Delete the entire `bias` function as this performs the same steps as the default from the base class.